pkg/validation/app.go

```diff
@@ -266,6 +283,19 @@ func (v *Validator) validateLabels(ctx context.Context, cr v1alpha1.App) error {
 	return nil
 }
 
+func (v *Validator) validateOrgLabels(ctx context.Context, cr v1alpha1.App) error {
+	// For org-namespaced App CRs make sure the `giantswarm.io/cluster` is set,
+	// and there is no conflicting `app-operator.giantswarm.io/version` label.
+	if key.ClusterLabel(cr) == "" {
+		return microerror.Maskf(validationError, labelNotFoundTemplate, label.Cluster)
+	}
+	if key.VersionLabel(cr) != "" {
+		return microerror.Maskf(validationError, labelsConflictTemplate, label.AppOperatorVersion, label.Cluster)
+	}
```

Comment:
This validation is correct but it's going to make migration tricky.

I think we should remove it until the version label is removed from `default-apps-openstack` and all clusters are using the latest version of the app.

Reply:
Sure, if the version label is harmless for org-namespaced apps then we can skip this part, I put it here because I assumed they both cannot or shouldn't coexist together.

Do you want me to remove this part completely (and thus the `labelsConflictTemplate`) or turn it into a a debug message saying something like `hey, you have a label that's not supported for org namespaced apps`?

Reply:
Yes they can coexist. What matters is which namespace app-operator is watching and the label selector its using. 
https://github.com/giantswarm/app-operator/blob/e308ca8cd6938c17f871916c596704f389242cc3/service/controller/app/app.go#L124_L140

What we want to avoid is 2 instances of app-operator processing the same app CRs.

Let's remove it completely. We can add it later if needed.

Reply:
Got it!